model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/SpanScalingRootCauseInsight.kt

```diff
@@ -0,0 +1,39 @@
+package org.digma.intellij.plugin.model.rest.insights
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import org.digma.intellij.plugin.model.InsightType
+import java.beans.ConstructorProperties
+import java.util.*
```

Comment:
lets get rid of this *
please follow this one for java:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

and for koltin use:
https://stackoverflow.com/questions/67341957/how-do-i-disable-wildcard-imports-for-all-kotlin-files-in-android-studio

Reply:
Done